src/System.Device.Gpio/System/Device/Gpio/Drivers/Windows10DriverPin.Windows.cs

```diff
@@ -141,7 +141,7 @@ void Handler(WinGpio.GpioPin s, WinGpio.GpioPinValueChangedEventArgs a)
 
                 return new WaitForEventResult
                 {
-                    EventTypes = pinEventTypes,
+                    EventTypes = eventOccurred ? pinEventTypes : PinEventTypes.None,
```

Comment:
Is this actually required? I see that we are already setting the PinEventTypes to none on l ine 121, so I believe that this extra check is not required.

Reply:
No, it was not required, of course.